RELEASE.md

```diff
@@ -1,11 +1,62 @@
 # Upcoming release:
 
-Add release notes for the upcoming release here.
+<!--
+Use the sections below to add notes for the next release.
+
+Please follow the established format:
+- Keep each note concise - ideally commit title length
+- Use present tense (e.g. 'Add new feature')
+- Include the ID number for the related PR (or PRs) in parentheses
+-->
+
+## Major features and improvements
+
+<!-- Add release notes for the upcoming release here -->
+
+## Bug fixes and other changes
+
+<!-- Add release notes for the upcoming release here -->
+
```

Comment:
I would remove this part, as we only want this upcoming release section in develop branch. We can have another PR to add this section after the release is complete.

Reply:
A) It doesn't matter to have this in the master branch, as it's commented out, so it's not visible in preview
B) I don't think we can maintain having different code in develop and master